lms/static/sass/views/_learner-profile.scss

```diff
@@ -307,8 +308,244 @@
             border: 2px dashed $link-color;
             .u-field-title,
             i {
-              color:  $link-color;
+                color: $link-color;
+            }
+        }
+    }
+
+    .badge-paging-header {
+        padding-top: $baseline;
+    }
+
+    .pagination {
```

Comment:
Also, has this pagination pattern been reviewed from an a11y PoV by @cptvitamin? I don't believe all of Studio's UI patterns have in the past.